.ci/validation.sh

```diff
@@ -712,7 +712,8 @@ no-error-equalsverifier)
   echo CS_version: ${CS_POM_VERSION}
   checkout_from https://github.com/jqno/equalsverifier.git
   cd .ci-temp/equalsverifier
-  mvn -e --no-transfer-progress compile checkstyle:check -Dcheckstyle.version=${CS_POM_VERSION}
+  mvn -e --no-transfer-progress --show-version -P-enables-javafx-tests-for-jdk11-and-up \
```

Comment:
Equalsverifier has a profile for Java 11+:
https://github.com/jqno/equalsverifier/blob/main/pom.xml#L352
```xml
        <profile>
            <id>enables-javafx-tests-for-jdk11-and-up</id>
            <activation>
                <jdk>[11,)</jdk>
            </activation>

            <dependencies>
                <dependency>
                    <groupId>org.openjfx</groupId>
                    <artifactId>javafx-base</artifactId>
                    <version>17.0.1</version>
                    <scope>provided</scope>
                </dependency>
            </dependencies>
```
This profile requires JavaFX. Installing JavaFX for ppc64le linux is not trivial, and it is not actually necessary, so this profile is disabled.

Reply:
Author of this repo told that we need jdk17  for static analysis tools
Do we have jdk17 in CI ?

Reply:
Yes, we even have jdk18 in Travis. I'll try to fix Equalsverifier in a separate PR

Reply:
> Do we have jdk17 in CI ?

Sorry, latest build for [AdoptOpenJdk](https://adoptopenjdk.net/installation.html) is jdk16.

Reply:
Note that the `enables-javafx-tests-for-jdk11-and-up` profile no longer exists, so you can remove that if you like.

Also, as I mentioned in https://github.com/jqno/equalsverifier/issues/586#issuecomment-1025459469, I can make it so that Checkstyle in the EqualsVerifier build will run on a JDK of your choice, just let me know in that ticket and I'll make it happen.